### DISCUSSION TITLE
Welcome to gnucash Discussions!

### DISCUSSION THREAD

**Discussion by DenissLarka** [DenissLarka/jgnucashlib#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RSHKwee** [comment]
I am a pensionado with extensive Pascal realtime programming on OpenVMS systems experience.

And now I am learning Java programming.
I maintain my cashflows with GnuCash.

And I stumbled on this library.

**Reply by RSHKwee** [reply]
Can you (Denis Larka) something about yourself?😉

**Comment by cfnz** [comment]
Hi, just looking at this library, I see the readme says

> As mentioned above: As of now, the lib only works well when your GnuCash files are generated on a german system (locale de_DE)

With my small amount of testing, it seems to work with English. Could you elaborate on what might not work?

**Reply by DenissLarka** [reply]
There should be no encoding-related issues, as all my tests have been conducted using UTF-8.
